src/quicr_client_raw_session.cpp

```diff
@@ -123,9 +123,10 @@ class QuicRTransportDelegate : public qtransport::ITransport::TransportDelegate
                                "Dropping malformed message: " +
                                  std::string(e.what()));
         return;
-      } catch (const std::exception& /* ex */) {
+      } catch (const std::exception& e) {
         client.log_handler.log(qtransport::LogLevel::info,
-                               "Dropping malformed message");
+                               "Dropping malformed message: " +
+                                 std::string(e.what()));
```

Comment:
While that might work, why not put `std::string()` around the C-string? Then the `operator+` will work. Not a big deal, if this gets converted properly.